pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -232,19 +223,15 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 		return nil, err
 	}
 
-	cfg.Auth = kubeletserver.NewKubeletAuth(authn, authzAttr, authz)
-
-	// Make sure the node doesn't think it is in standalone mode
-	// This is required for the node to enforce nodeSelectors on pods, to set hostIP on pod status updates, etc
-	cfg.StandaloneMode = false
```

Comment:
@smarterclayton confirm that you checked and this is no longer required for nodeSelectors to work.